osu.Game/Overlays/MusicController.cs

```diff
@@ -260,10 +270,19 @@ public bool OnPressed(GlobalAction action)
                     return true;
 
                 case GlobalAction.MusicPrev:
-                    if (PrevTrack())
-                        onScreenDisplay?.Display(new MusicControllerToast("Previous track"));
-
-                    return true;
+                    switch (PrevTrack())
+                    {
+                        case PreviousButtonAction.Restart:
+                            onScreenDisplay?.Display(new MusicControllerToast("Restart track"));
+                            return true;
+
+                        case PreviousButtonAction.Previous:
+                            onScreenDisplay?.Display(new MusicControllerToast("Previous track"));
+                            return true;
+
+                        default:
+                            return false;
```

Comment:
Should return `true` instead - the return value indicates whether the action is handled or not, and it is, even if it does nothing (see the neighboring cases - `return true` is done unconditionally, disregarding whether the hotkeys did anything or not). Therefore I'd probably move the `return`s outside, and not even handle `default` explicitly.